### ISSUE TITLE
Comparison to bincode?

### ISSUE THREAD

**Issue by evbo** [vigna/epserde-rs#29]
The readme introduces the problem so well, but you don't mention bincode, which also aims to solve the problem of binary storage of large structures from memory.

What is epserde-rs better fit for vs existing giants in this space like bincode?

**Comment by zommiommy** [comment]
Yeah we should mention it, bincode while it's a great crate, it doesn't do zero copy deserialisation which is the core aspect of epserde. 

Moreover, it doesn't have any file header / versioning, in epserde we went to great lengths to hash the serialized type data layout. This way if the user changes the order of fields in a repr(C) struct or changes a type, epserde will complain and refuse to load an incompatible file.
